calico_node/tests/st/utils/docker_host.py

```diff
@@ -186,7 +186,7 @@ def __init__(self, name, start_calico=True, dind=True,
         if start_calico:
             self.start_calico_node()
 
-    def execute(self, command, raise_exception_on_failure=True):
+    def execute(self, command, raise_exception_on_failure=True, daemon_mode=False):
         """
         Pass a command into a host container.
```

Comment:
Should probably explain what daemon_mode is and why you'd use it?

Reply:
Comments added.